packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -457,11 +500,43 @@ function MultiplierOption({
   );
 }
 
+function InOutOption({
+  inOutMode,
+  outValue,
+  disabled,
+  onToggle,
+  onChangeText,
+}) {
+  return (
+    <View style={{ flexDirection: 'row', gap: 10, height: 28 }}>
+      <CheckboxOption
+        id="form_inout"
+        checked={inOutMode}
+        disabled={disabled}
+        onChange={onToggle}
+      >
+        {inOutMode
+          ? 'in/out field'
+          : 'Select column to specify if amount goes in/out'}
+      </CheckboxOption>
+      {inOutMode && (
+        <Input
+          type="text"
+          value={outValue}
+          onUpdate={onChangeText}
+          placeholder="Out value"
```

Comment:
```suggestion
          placeholder="Value used to tell that this row is a OUT transaction (i.e. Credit)"
```

Open to other suggestions here too. But I definitely think we should work on the original wording to make it more explicit

Reply:
Had to make sure the placeholder actually fits the input field. Hope this is clear enough, I think the "i.e. Credit" helps a lot.
![image](https://github.com/actualbudget/actual/assets/21026046/bcfe3724-d855-4d7d-b2fe-e906a010e6d0)